
<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->


<meta charset="utf-8">



<!-- begin SEO -->









<title>Sitemap - Homepage</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Homepage">
<meta property="og:title" content="Sitemap">


  <link rel="canonical" href="http://localhost:4000/sitemap/">
  <meta property="og:url" content="http://localhost:4000/sitemap/">





  <meta name="twitter:site" content="@Rocha_Lab">
  <meta name="twitter:title" content="Sitemap">
  <meta name="twitter:description" content="enthusiastic researcher">
  <meta name="twitter:url" content="http://localhost:4000/sitemap/">

  
    <meta name="twitter:card" content="summary">
    
  

  



  

  












  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Susana Rocha's Lab",
      "url" : "http://localhost:4000",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="http://localhost:4000/feed.xml" type="application/atom+xml" rel="alternate" title="Homepage Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="http://localhost:4000/assets/css/main.css">

<meta http-equiv="cleartype" content="on">

    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="http://localhost:4000/images/apple-touch-icon-57x57.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="60x60" href="http://localhost:4000/images/apple-touch-icon-60x60.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="72x72" href="http://localhost:4000/images/apple-touch-icon-72x72.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="76x76" href="http://localhost:4000/images/apple-touch-icon-76x76.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="114x114" href="http://localhost:4000/images/apple-touch-icon-114x114.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="120x120" href="http://localhost:4000/images/apple-touch-icon-120x120.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="144x144" href="http://localhost:4000/images/apple-touch-icon-144x144.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="152x152" href="http://localhost:4000/images/apple-touch-icon-152x152.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="180x180" href="http://localhost:4000/images/apple-touch-icon-180x180.png?v=M44lzPylqQ">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-32x32.png?v=M44lzPylqQ" sizes="32x32">
<link rel="icon" type="image/png" href="http://localhost:4000/images/android-chrome-192x192.png?v=M44lzPylqQ" sizes="192x192">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-96x96.png?v=M44lzPylqQ" sizes="96x96">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-16x16.png?v=M44lzPylqQ" sizes="16x16">
<link rel="manifest" href="http://localhost:4000/images/manifest.json?v=M44lzPylqQ">
<link rel="mask-icon" href="http://localhost:4000/images/safari-pinned-tab.svg?v=M44lzPylqQ" color="#000000">
<link rel="shortcut icon" href="/images/favicon.ico?v=M44lzPylqQ">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-TileImage" content="http://localhost:4000/images/mstile-144x144.png?v=M44lzPylqQ">
<meta name="msapplication-config" content="http://localhost:4000/images/browserconfig.xml?v=M44lzPylqQ">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="http://localhost:4000/assets/css/academicons.css"/>
<!-- end custom head snippets -->

    <!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    
<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->


<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="http://localhost:4000/">Homepage</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/news/">News</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/outreach/">Science Outreach</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/projects/">Projects</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/team/">Research team</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/talks/">Talks & Posters</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/positions/">Positions</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    



<div id="main" role="main">
  


  <div class="sidebar sticky">
  



<div itemscope itemtype="http://schema.org/Person">

  <div class="author__avatar">
    
    	<img src="http://localhost:4000/images/profile.png" class="author__avatar" alt="Susana Rocha's Lab">
    
  </div>

  <div class="author__content">
    <h3 class="author__name">Susana Rocha's Lab</h3>
    <p class="author__bio">Imaging the forces driving cells</p>
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> KU Leuven, Belgium</li>
      
      
      
      
        <li><a href="mailto:susana.rocha@kuleuven.be"><i class="fa fa-fw fa-envelope-square" aria-hidden="true"></i> Email</a></li>
      
      
        <li><a href="https://scholar.google.be/citations?user=OsiAwIYAAAAJ&hl=en&citsig=AMstHGQQx0uqfgCvrTEny2ghp98R3FtD5Q"><i class="ai ai-google-scholar-square ai-fw"></i> Google Scholar</a></li>
      
      
        <li><a href="https://orcid.org/0000-0003-1258-9396"><i class="ai ai-orcid-square ai-fw"></i> ORCID</a></li>
      
      
      
        <li><a href="https://www.researchgate.net/profile/Susana_Rocha9"><i class="fa fa-fw fa-book" aria-hidden="true"></i> Research Gate </a></li>
      
      
      
        <li><a href="https://twitter.com/Rocha_Lab"><i class="fa fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
      
      
      
      
        <li><a href="https://www.linkedin.com/in/susana-rocha-50020622"><i class="fa fa-fw fa-linkedin-square" aria-hidden="true"></i> LinkedIn</a></li>
      
      
      
      
      
      
        <li><a href="https://github.com/susanarocha"><i class="fa fa-fw fa-github" aria-hidden="true"></i> Github</a></li>
      
      
      
      
      
      
      
      
      
      
      
      
      
      <li><a href="http://susanarocha.github.io/files/CV.pdf">CV Sep2018</a></li>
    </ul>
  </div>
</div>

  
  </div>


  <div class="archive">
    
      <h1 class="page__title">Sitemap</h1>
    
    
<p>A list of all the posts and pages found on the site. For you robots out there is an <a href="http://localhost:4000/sitemap.xml">XML version</a> available for digesting as well.</p>

<h2>Pages</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/404.html" rel="permalink">Page Not Found</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Page not found. Your pixels are in another canvas.</p>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/" rel="permalink"></a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>About me</p>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/archive-layout-with-content/" rel="permalink">Archive Layout with Content</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/categories/" rel="permalink">Posts by Category</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/collection-archive/" rel="permalink">Posts by Collection</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/assets/css/main.css" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/" rel="permalink">News</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/non-menu-page/" rel="permalink">Page not in menu</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>This is a page not in th emain menu</p>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/outreach/" rel="permalink">Science Outreach</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/positions/" rel="permalink">Open Positions</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/" rel="permalink">Projects</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/" rel="permalink">Publications</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/sitemap/" rel="permalink">Sitemap</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/tags/" rel="permalink">Posts by Tags</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/talks/" rel="permalink">Talks &amp; Posters</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/" rel="permalink">Research Team</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/techniques/" rel="permalink">Techniques</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/terms/" rel="permalink">Terms and Privacy Policy</a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/about/" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/about.html" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/nmp/" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/nmp.html" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/sitemap.xml" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/robots.txt" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/feed.xml" rel="permalink"></a>
      
    </h2>

    



    

    

    

  </article>
</div>

<h2>Posts</h2>

<h2>bookchapters</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/bookchapters/2013-08-30-bookchapter2/" rel="permalink">Photoswitchable Fluorescent Proteins for Superresolution Fluorescence Microscopy Circumventing the Diffraction Limit of Light
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2013-08-30T00:00:00+02:00">August 30, 2013</time></p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=5909833811978467751&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/bookchapters/2016-11-03-bookchapter3/" rel="permalink">Distribution, Organization, and Dynamics of EGF Receptor in the Plasma Membrane Studied by Super-Resolution Imaging
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2016-11-03T00:00:00+01:00">November 03, 2016</time></p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=15168308514523697511&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<h2>mainoutreach</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/mainoutreach/cellmech/" rel="permalink">The matrix supporting life
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> From cells to tissues </h2>

</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="1902-01-01T00:00:00+00:00">January 01, 1902</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/mainoutreach/drugdelivery/" rel="permalink">Better ways to give medicines
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Cancer is one of the major causes of death in our society. The anti-cancer drugs currently used in chemotherapy produce several side effects and, although fatal for almost all cells in a tumor, a small percentage of cells appear to resist the treatment. It is therefore urgent to design new therapies that specifically target these cells while causing no harm to healthy tissue.
The resistance to multiple drugs is linked to the presence of specific molecules at the cellular plasma membrane, that actively pump out chemical drugs (efflux pumps). Additionally, some cancer cells have the ability to self-renew, thereby initiating secondary tumor growth. Multi-drug resistant cells and cancer stem cells are suggested to be the main cause of treatment failure and relapse.

</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="1902-01-01T00:00:00+00:00">January 01, 1902</time></p>
    

    

  </article>
</div>

<h2>maintech</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/maintech/organization/" rel="permalink">Molecular Organization at the Nanoscale
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description">
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="1901-01-01T00:00:00+01:00">January 01, 1901</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/maintech/dynamics/" rel="permalink">Molecular Dynamics and Protein Interactions
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>bla bla bla</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="1902-01-01T00:00:00+00:00">January 01, 1902</time></p>
    

    

  </article>
</div>

<h2>news</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-01-01-site-online/" rel="permalink">Site is online!
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>A website to share the cool science we do at KU Leuven.</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-01T00:00:00+01:00">December 01, 2017</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2017-12-19-ASCNano%20online/" rel="permalink">Correlative AFM / Fluorescence: Article online
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Congragulations <strong>Wout &amp; Willem</strong>!
<img src="/images/AFM_correlativeACSNano.gif" style="width: 30%" align="right" />
<br />
Nice article reporting new development in correlative AFM and Fluorescence imaging, and its application to study the interaction between DNA and DNA-interacting proteins.
The full article can be found  <a href="http://pubs.acs.org/doi/abs/10.1021/acsnano.7b05405"><u>here</u></a>.
<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-12-21T00:00:00+01:00">December 21, 2017</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-01-24-EMBOaward/" rel="permalink">EMBO travel grant awarded
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Congragulations <strong>Max</strong>!
<img src="/images/Max.png" style="width: 10%" align="right" />
<br />
Finally some good news from the EMBO organization: Max received the EMBO travel grant for the 3-month stay in Leuven.
He will be working on imaging focal adhesions in 3D (more information on this project can be found <a href="/projects/cell-adhesion"><u>here</u></a>)
<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-01-24T00:00:00+01:00">January 24, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-02-12-chemphotochem%20online/" rel="permalink">Conversion of Alexa Fluor 647: Article online
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Congragulations <strong>Lieve &amp; Koen</strong>! <br />
<img src="/images/photoconver647.jpg" style="width: 30%" align="right" />
<br />
After some time, this story is (finally) out. A nice collaboration between virology, microscopy and photophysics experts, reporting a phenomena that will give a lot to talk about…
The full article can be found  <a href="http://onlinelibrary.wiley.com/doi/10.1002/cptc.201700216/full"><u>here</u></a>.
<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-02-12T00:00:00+01:00">February 12, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-05-13_review%20virus/" rel="permalink">Review on imaging single virus particles is now online!
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Congragulations <strong>Nagma, Doortje &amp; Jelle</strong>! <br />
We were able to write a nice article reviewing our work in the virus field.
The full article can be found  <a href="http://www.mdpi.com/1999-4915/10/5/250"><u>here</u></a>.
<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-05-11T00:00:00+02:00">May 11, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-06-27%20FWO%20indra/" rel="permalink">FWO grant for Indra &amp; Boris
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Congragulations <strong>Indra &amp; Boris</strong>!
<img src="/images/nanogirls.png" style="width: 15%" align="right" />
<br />
Awarded FWO doctoral grant. It is going to be 4 years of fun with Nanoparticles &amp; Polymers!
<br />
(Unfortunately, I do not have such a pretty foto of Boris…)
<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-06-27T00:00:00+02:00">June 27, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-09-06-nature-online/" rel="permalink">Role of glutamine synthetase in angiogenesis beyond glutamine synthesis: Article online
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Congragulations <strong>Guy</strong>!
More than 2 years after submission, this story is finally out! I am proud of my small contribution.
The full article can be found  <a href="https://www.nature.com/articles/s41586-018-0466-7#article-info"><u>here</u></a>.
<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-09-06T00:00:00+02:00">September 06, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-09-09%20Mapping%20PPIs/" rel="permalink">Mapping Interactions at the Nanoscale: Article online
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Congragulations <b>Herlinde</b>!
<img src="/images/TOC2-01.png" style="width: 50%" align="right" />
<br />
I am very proud of this one. After all the struggle with the cloning, and all the re-writing to make the best story possible, 2 years after the PhD - it is out!
With the final contribution of Dr. Rafael Camacho and his nice simulations.
The full article can be found  <a href="https://pubs.acs.org/doi/10.1021/acsnano.8b01227 "><u>here</u></a>.<br />
The software to run the simulations can be found <a href="https://github.com/CamachoDejay/diffusion_binding_simulations"><u>here</u></a>.<br />
<br />
</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-09-09T00:00:00+02:00">September 09, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-11-21-EMBO%20madrid/" rel="permalink">Madrid here I come!
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Colaboration Leuven - Madrid keeps increasing!
<img src="/images/spacial_proteo.png" style="width: 17%" align="right" />
<br />
<br />
Thanks to <strong>EMBO</strong> I will be able to join Guillermo Solis and Dr. Rodrigo Barderas at the Instituto Salud D.Carlos III in Madrid.
I am really looking forward to meet the Barderas’ group and get this project going!
<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-11-21T00:00:00+01:00">November 21, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2018-12-19-Xmas2018/" rel="permalink">Best wishes for 2019
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p><img src="/images/xmas2018.jpg" style="width: 30%" align="right" />
<br /><br />
After an unforgettable Xmas party, a few days to (try to) finish up things and we are off to a well deserved Xmas break!
2019 might bring big changes - let’s hope they are all for the better!
<br />
And if not, let’s just continue the spirit and <strong>do great science!</strong>
<br /><br />
<br />
Happy new year!
<br /><br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-12-19T00:00:00+01:00">December 19, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2019-01-17-article%20Max/" rel="permalink">Synthetic extracellular matrices regulate cellular organization: First article of Max!
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p><img src="/images/articleMax.png" style="width: 40%" align="right" />
Even though we were not directly involved, the Leuven team is very happy to see this work published.
Congratulations <strong>Max</strong>! All your hard work is starting to pay off. This is the first one, hopefully many more will follow.
<br />
The full publication can be found in the <a href="https://pubs.acs.org/doi/10.1021/acs.biomac.8b01445"><u>journal</u></a> or at <a href="https://www.researchgate.net/publication/330146907_Synthetic_extracellular_matrices_with_nonlinear_elasticity_regulate_cellular_organization"><u>Research gate</u></a>.</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-17T00:00:00+01:00">January 17, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2019-02-13-MarieCurie%20Hongbo/" rel="permalink">Marie Curie Individual Fellowship for Dr. Hongbo Yuan
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Congratulations <strong>Hongbo</strong>!
<br />
Dr. Hongbo Yuan will be joining us after the summer, to investigate forces from the cell or the ECM change the mechanical properties of the matrix and, more importantly, how this change affects biological functions. We are really looking forward to work with you on this.
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-02-13T00:00:00+01:00">February 13, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2019-02-25-article%20bea/" rel="permalink">Highly Efficient Multifunctional Drug Delivery Systems: Article Online!
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p><img src="/images/article_bea.png" style="width: 30%" align="right" />
A nice article showing how simple chemistry can improve cancer specificity and efficiency of drug delivery systems. <br />
Congratulations <strong>Bea</strong>!
<br />
You were also able to (finally) show that PEI does induce endossomal escape. And, for the non-believers, more evidence will follow!
<br />
The full publication can be found in the <a href="https://www.nature.com/articles/s41598-019-39107-3"><u>journal</u></a> or at <a href="https://www.researchgate.net/publication/331326850_Polymeric_Engineering_of_Nanoparticles_for_Highly_Efficient_Multifunctional_Drug_Delivery_Systems"><u>Research gate</u></a>.</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-02-25T00:00:00+01:00">February 25, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2019-06-06-caps%20arrived/" rel="permalink">Lab caps arrived!
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p><img src="/images/labcap.png" style="width: 20%" align="right" />
We could have ordered pens or mugs but… the amount of sun in Belgium requires protection! <br />
An item to give to current and future lab members.
<br />
<em>(now we have to wait for the end of the summer to join the team for a group picture)</em>
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-06-06T00:00:00+02:00">June 06, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2019-06-18-visit%20Lx/" rel="permalink">Visit to the Technical University of Lisbon
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p><img src="/images/lisbon20192.jpg" style="width: 30%" align="right" />
Thanks to Dr. Nuno Canha, I could present my research to colleagues from my home country, including some good friends.
<br />
It was a short visit, but a memorable one. <br />
The quality of the science being done in countries with very limited resources will always amaze me. For good science, you need a good brain, with a substantial amount of creativity.<br />
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-06-18T00:00:00+02:00">June 18, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/news/2019-07-08-core2core/" rel="permalink">Leuven-Hokkaido collaboration: kickoff meeting
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p><b> Core-to-core meeting </b>
<img src="/images/posterguys-01.jpg" />
<br /><br />
Today was a day of many fruitful discussions with visitors from Hokkaido University. <br />
I had the opportunity to give a talk presenting our research, while Johannes and Quinten could present their progress during the poster session. <br />
We have found some common interests in fluorescence-based sensors, drug nano delivery systems and extracellular matrix. Next meeting in Japan - already looking forward!
<br /></p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-07-08T00:00:00+02:00">July 08, 2019</time></p>
    

    

  </article>
</div>

<h2>outreach</h2>

<h2>projects</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/struct-matrix/" rel="permalink">Structure of Biomimetic Materials
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>A large number of natural and synthetic hydrogels are currently used for tissue engineering and regenerative medicine. Over the last decade, there has been an increasing awareness of the role of material properties of the substrates in guiding cellular behaviour. This has inspired chemists to create a new generation of materials with mechanical properties closed to that of natural occurring biopolymer networks. Recently, the groups of <b>Prof. Alan Rowan </b>(Queens University, Australia) and <b>Prof. Paul Kouwer</b> (Radboud University of Nijmegen, The Netherlands) were able to develop a fully synthetic material that mimics in all aspects the gels prepared from cellular filaments. These synthetics gels are prepared from <strong>polyisocyanopeptides (PICs)</strong> grafted with oligo(ethylene glycol) chains and share structural features of biopolymers: their helical structure renders the polymer molecules relatively stiff while the interaction between the side chains enable the formation of bundles or fibrils of defined dimensions. The triethylene glycol side chains attached to the polymer backbone render the material thermo-responsive (it will gel upon heating beyond 20 °C and become liquid again upon cooling). Despite being characterized extensively in bulk, the fundamental dynamics and the relation between the macroscopic properties and the microscopic structure at cellular length scales of PIC-based hydrogels remains obscure.
<br /></p>
<figure style="width: 50%" class="align-center">
<img src="/images/PIC_struct-01.png" />
<figcaption>Structure of the PIC polymer/monomer unit and cartoon of the polymer structure showing the helical structure.</figcaption>
</figure>
<p>Classically, structural characterization of materials is performed with electron microscopy or scanning probe microscopy. Despite the high spatial resolution achievable with these techniques, they are unable to measure dynamics ‘in situ’ and sample preparation can be a laborious process. In contrast, optical microscopy has the potential to unravel the dynamics in complex heterogeneous systems but has been limited to a spatial resolution of ca. 200 nm. In the past 10 years fluorescence imaging has been revolutionized by the successful development of sub-diffraction (super-resolution) microscopy modalities which can achieve resolutions down to tens of nanometers (see
<a href="/maintech/organization">Molecular Organization at the Nanoscale</a>).The various possibilities of fluorescence microscopy to probe dynamics and heterogeneities, with molecular resolution, for a wide range of time scales makes it an ideal tool to address many topics of polymer science.
In this project we are using STED to image the polymer network at the nanometer scale.
<br /></p>
<figure style="width: 50%" class="align-center">
<img src="/images/PIC-STED.jpg" />
<figcaption>STED image of PIC network. </figcaption>
</figure>
<p><br />
For more information on PIC-based hydrogels:</p>
<ul>
  <li>Kouwer P.H.J., et al. (2013) <em>Responsive biomimetic networks from polyisocyanopeptide hydrogels</em>, Nature, 493, pages 651–655 (article can be found <a href="https://www.nature.com/articles/nature11839">here</a>)</li>
  <li>Jasper M., et al. (2014) <em>Ultra-responsive soft matter from strain-stiffening hydrogels</em>, Nature Communications, 5, 5808 (article can be found <a href="https://www.nature.com/articles/ncomms6808">here</a>)</li>
  <li>Jasper M., et al. (2016) <em>Bundle Formation in Biomimetic Hydrogels</em>, Macromolecules, 17(8), pages 2642–2649 (article can be found <a href="https://pubs.acs.org/doi/abs/10.1021/acs.biomac.6b00703">here</a>)</li>
</ul>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/reptation/" rel="permalink">Polymer reptation in 3D
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Our current theoretical understanding of entangled polymer chain dynamics is based on the reptation model. First proposed by Doi and Edwards, and further expanded by de Gennes, the reptation model assumes that a polymer chain is confined by the surrounding matrix and is therefore forced to move inside an imaginary tube defined by the transient network of entangled neighboring chains. Intuitively this motion resembles that of a snake or worm.
The reptation model predicts five dynamical regimes for segment diffusion, summarized in the figure below. These regimes are as follows: (0) sub-segmental processes (“glassy dynamics”) at very short times (microseconds), (I) small motion subject only to chain connectivity, (II) “local reptation”: short-distance motion within the constraints imposed by the surrounding chains (“tube”), (III) “reptation”: diffusive motion along the curvilinear tube over distances larger than the polymer size, and (IV) free diffusion.
<br /><br />
<img src="/images/PIC_WF.jpg" /></p>

</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/rheology/" rel="permalink">Rheology at the micrometer scale
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Due to the crucial role of physical cues in regulating cell behaviour, the mechanical properties of hydrogels are a key design parameter in tissue engineering applications. The shear elastic properties of viscoelastic materials are commonly measured by mechanical rheometers. Storage and loss moduli of a material can be measured by application of strain while measuring stress or vice versa. In contrast, recently developed optical micro-rheology techniques use nanometer- or micrometer-sized particles embedded in the material to obtain the viscoelastic response parameters. Thermal or passive micro-rheology for viscoelastic materials is based on an extension of the concepts of Brownian motion of particles in simple liquids.
The movement of the embedded particles can be monitored using particle tracking. Initially developed to investigate the rheological properties of uniform complex fluids, particle tracking micro-rheology (PTM) is becoming a popular technique to analyze polymer blends and gels, as well as the deformability and elasticity within cells. However, if the beads locally modify the structure of the gel or are contained in a pore in an inhomogeneous matrix, the bulk rheological properties will not be retrieved. A solution is to use the cross-correlated thermal fluctuation of pairs of tracer particles, ‘two-point micro-rheology’. This method provides a better agreement between micro and macro-rheology, even in complex micro-structured fluids. However, technical constrains limit the wide application of this technique. One of the major limitations of two-point micro-rheology is the reduced number of trajectories that can be used for analysis. During particle tracking micro-rheology, the length of the calculated trajectories is limited by the time spent by the tracers in the field of view (x,y) and depth of focus (z). Consequently, mechanical characterization of complex polymer matrixes at the micrometer scale would benefit greatly of a new method for (fast) tracking in 3D.
We are developing a new method for fast tracking of (fluorescent) beads in 3D using a multi-plane wide field microscope. This will allow a better mechanical characterization of soft materials, at the microscale.</p>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/cell-adhesion/" rel="permalink">Cellular adhesion in 3D matrices
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Cells sense physical forces and the mechanical properties of the microenvironment via several distinct mechanisms and cellular components. The first step of cellular adhesion to the ECM occurs via transmembrane heterodimers of the integrin family.  Once integrin molecules adhere to the ECM, they are activated and form clusters.  As the number of bound molecules increases, some of the focal complexes evolve from small (0.5-1µm in diameter) transient ‘dot-like’ contacts to elongated structures (3-10µm) which couple with actin and associated proteins. The mechanical coupling between the ECM and the cell cytoskeleton is controlled by the dynamics of the focal adhesion complexes (assembly, disassembly and turnover).</p>

</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/PPIs/" rel="permalink">Protein-Protein Interactions
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Protein-protein interactions (PPIs) are intrinsic to all cellular processes, driving both metabolic and regulatory pathways. Despite the numerous techniques available, detection of transient short-lived PPIs remains challenging4. The main fluorescence microscopic techniques developed for visualizing PPIs in a cellular context are based on Föster resonance energy transfer (FRET) or bimolecular fluorescence complementation (BiFC)5. Both techniques detect the interaction between a pair of labeled molecules. Although highly informative, they require fine positioning of the labels and in the majority of the applications the spatial resolution achieved is limited by the diffraction of light to about 200 nm.
<i>More information concerning the use of FRET to detect PPIs can be found at <a href="/projects/cell-signal">Cellular Signalling</a>).</i></p>
<p>
We have use a single molecule localization based super-resolution technique to detect and map PPIs at the cell membrane. This new variant of PAINT that enables mapping of short-lived transient interactions between cytosolic and membrane-bound proteins inside living mammalian cells, at the nanometer scale. In this method the protein of interest is labeled with a light-controllable fluorescent protein and imaged under TIRF illumination, which leads to the selective activation and subsequent detection of molecules in close proximity with the plasma membrane. Interacting molecules are discriminated using a stringent fitting of the fluorescence signal recorded for every single molecule.

</p>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/empty/" rel="permalink">
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description">
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/organoids/" rel="permalink">Organoids: models for cell communication
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Nowadays, human organoids are becoming a highly promising tool to model organ development, function and especially human diseases in vitro. In general, organoids are miniature, simplified organs that can easily propagate in vitro originating from one or a few cells, typically stem cells.</p>

</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/endoscopy/" rel="permalink">Single cell manipulation by endoscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Nanowire-based endoscopy has attracted interest due to its ability to manipulate cells at the single-cell level with minimal cellular perturbation. High-density, vertically aligned nanowire arrays have been used as an efficient gene delivery system. Despite the high transfection rates, culturing the cells on nanowire arrays might have other influences on the cellular behaviour. For example, stem cells cultured on silicon nanowires show significantly different adhesion, proliferation and differentiation, compared with flat silicon or other control substrates. Furthermore, such arrays are not location-specific and require optimization of the nanowire density and dimension for the different the cell types. In collaboration with the group of Prof. Hiroshi Uji-i we are developing a method to delivery genetic material using a single nanowire. In contrast to the existing methods, this approach can be applied to any cell type and is extremely specific: it can target a single cell and it can deliver the genetic material exactly at the desired position, such as inside of the nucleus, with no damage to the cell. Since gene editing is a stochastic event occurring in only a fraction of the cells, the transfer of genetic material (or proteins) is of crucial importance in genome editing methods, where the nucleases must be efficiently delivered. The duration and magnitude of the nuclease expression are critical parameters for the level of both on-target and off-target nuclease activity. Additionally, the dose of donor template DNA is important to ensure efficient homologous recombination. The proposed method offers the possibility to deliver different molecules at different times, in synchronization with the cell cycle.
The lab of Prof. Uji-i is one of the first (and few) groups worldwide to have developed and optimized a novel nanoscopic technique using 1D nanowires, with a diameter of less than 100 nm, for SERS endoscopic studies. It has been already proven by us that the thin diameter and 1D structure of the NW greatly reduces the damage induced to a live cell during probe insertion. Although designed for a different purpose, this nanoprobe is ideal as a starting point to develop a new NW-based gene delivery system.</p>
<figure style="width: 80%" class="align-center">
<img src="/images/endoscopy.png" />
<figcaption> Principle of nanowire-based gene delivery system. </figcaption>
</figure>

</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/DDS/" rel="permalink">New Drug delivery systems
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>In this decade, the pharmacology field has been intensively exploring different approaches to deliver multiple drugs with a single <strong>drug nano-carrier</strong>, such as liposomes, polymer nanoparticles, and inorganic nanoparticles. The advantage of nanoparticle based drug delivery is the ability to unify pharmacokinetics by simultaneous delivery of multiple drugs to specific target cells. <br />
<br />
Ever since first reported in 2001, mesoporous silica nanoparticles (MSNPs) have manifested themselves as highly potential candidates for targeted drug delivery. They owe their popularity to their high drug load capacity, chemical stability, biocompatibility and easy functionalization. Since the diameter of the nanoparticles (100 to 200 nm) is tunable, one can obtain a size suitable for passive targeting through the hyperpermeable tumor vasculature, thereby promoting accumulation of the nanoparticles in tumor tissue due to the enhanced permeability and retention effect (EPR). Additionally, functionalization of the nanoparticles with ligands which have a high affinity for tumor cell specific surface receptors promotes more specific internalization in cancer cells. For example, hyaluronic acid (HA) has been extensively used as a targeting ligand due to its affinity for CD44, a transmembrane glycoprotein receptor that plays a critical role in malignant cell activities and, most importantly, it is overexpressed in many solid tumor cells, in metastasis and cancer stem cells.</p>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/FluoAFM/" rel="permalink">Correlative AFM and Fluorescence Microscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Biological processes are often carried out in the context of macromolecular assemblies. In addition, arrangements of these complexes can be dynamic, resulting in a heterogeneous ensemble. Single molecule techniques can resolve distinct populations in heterogeneous systems, in contrast to bulk experiments where heterogeneity is averaged out. In turn, mechanistic details of bio-macromolecular interactions can be uncovered.
Atomic force microscopy (AFM) is a technique that can generate 3D reconstructions of individual biomolecules and complexes thereof in a label-free fashion, and with ~ nm resolution. To this end a very sharp tip, mounted on a flexible cantilever, scans a sample surface in a raster pattern using a piezo-scanner, while keeping the interaction force between sample and tip constant. In every pixel (x,y) of the scanned area, the z-position is recorded. Consequently, a 3D representation of the surface topography can be reconstructed.
An alternative way to study single molecules is by fluorescence microscopy. The molecule of interest is labeled with a fluorescent tag providing high contrast. Emission of the tag after excitation, is detected through an optical system. Due to the wave character of light, the emitted light is spread out on the detector described by the point spread function (PSF) of the optical system. This effect limits the resolution achieved with optical microscopy, referred to as the diffraction limit. However, when the signal of a single molecule is detected, the position of this molecule can be determined by fitting of the recorded fluorescence signal with a mathematical approximation of the PSF such as a two-dimensional Gaussian function. This principle underlies single molecule localization microscopy (SMLM).
AFM and SMLM are highly complementary technologies: AFM can provide insight in topographic features at a nanometer resolution while SMLM is sensitive towards specifically labelled molecules in complex samples. Integrated setups combining both technologies can therefore provide orthogonal information at the single-molecule level.</p>

</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/cell-signal/" rel="permalink">Cell signalling: probes and methods
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Cell signaling involves the sensing of an extracellular signal by a cell surface receptor, which then transduces this signal to an intracellular response. Despite the numerous studies performed on signaling pathways and mechanisms, little is known about the initial steps occurring at the plasma membrane: receptor pre-assembly at the molecular level and potential reorganization after ligand activation. Traditionally crystallography is used to investigate receptor multimerization. However, the crystallized state might not represent the biochemically active form due to the harsh preparation conditions and the absence of the cellular environment. Other approaches include macroscopic biochemical or biophysical methods, such as chemical cross-linking, ion-channel gating, immunoprecipitation or binding assays. Nowadays, established fluorescence imaging and spectroscopic techniques offer a versatile toolbox to study membrane receptor organization in (living) cells.
<br /><br />
In the lab we are using fluorescence fluctuation spectroscopy to quantify physicochemical processes (mobility, binding affinity, stoichiometry, absolute concentration) occurring on a micro-to-millisecond time scale. Fluorescence experiments down to picoseconds are also commonly possible with methods such as time-correlated single photon counting (TCSPC), that allow, e.g., measuring fluorescence lifetimes and molecular tumbling.  Additionally, spatially resolved microscopy with high temporal resolution also has clear benefits. For example, combined with confocal laser scanning microscopy (LSM), TCSPC allows protein-protein interactions (PPIs) to be imaged via Förster resonance energy transfer (FRET) based fluorescence lifetime imaging microscopy (FLIM). Imaging based FCS methods such as raster (RICS), number and brightness analysis (N&amp;B) or (spatio-) temporal image correlation spectroscopy [(S)TICS] combine the quantitative analytical power of fluctuation methods with spatial information to map, among many other things, mobility and stoichiometry inside living systems. Simultaneous dual-color fluorescence imaging is possible when fast alternating excitation (alias pulsed interleaved excitation, PIE) is employed. PIE renders analysis of dual-color point FCS experiments considerably more straightforward. The combination of PIE with fluctuation imaging (PIE-FI) allows extracting the maximum amount of molecular information (mobility, stoichiometry, interactions…) from each species present in dual-color LSM images.</p>
<figure style="width: 100%" class="align-center">
<img src="/images/pie.png" />
<figcaption>PIE (a), PIE-FI (b) and subsequent analyses, based on spatial/temporal auto-/cross-correlation or fluorescence lifetimes, which allow to extract the maximum amount of information of the molecules present in the imaged structure.</figcaption>
</figure>
<p>For more information on these methods:</p>
<ul>
  <li>Hendrix J., Lamb D.C. (2014) <em>Implementation and Application of Pulsed Interleaved Excitation for Dual-Color FCS and RICS.</em> In: Engelborghs Y., Visser A. (eds) Fluorescence Spectroscopy and Microscopy. Methods in Molecular Biology (Methods and Protocols), vol 1076. Humana Press, Totowa, NJ (chapter can be found <a href="https://link.springer.com/protocol/10.1007%2F978-1-62703-649-8_30"><u>here</u></a>)</li>
  <li>Hendrix J., Schrimpf W., Höller M., Lamb D.C. (2013) <em>Pulsed Interleaved Excitation Fluctuation Imaging</em>, Biophysical Journal, 105(4), 848-861 (article can be found <a href="https://www.sciencedirect.com/science/article/pii/S0006349513006863?via%3Dihub"><u>here</u></a>)
<br /><br /></li>
</ul>
</p>
    

    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/projects/HIV/" rel="permalink">Imaging single HIV virions
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>Viruses are simple agents exhibiting complex reproductive mechanisms. Decades of research have provided crucial basic insights, antiviral medication and moderately successful gene therapy trials. The most infectious viral particle is, however, not always the most abundant one in a population, questioning the utility of classic ensemble-averaging virology. Indeed, viral replication is often not particularly efficient, prone to errors or containing parallel routes.
In collaboration with Prof. Zeger Debeyser (KU Leuven) and Prof Hendrix (UHasselt) we have applied different single-molecule sensitive fluorescence methods to investigate viruses, one-by-one. While this collaboration is still ongoing, there is already several publications that show-case the potential of imaging single virions.</p>

</p>
    

    

    

  </article>
</div>

<h2>publications</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2004-10-01-paper1/" rel="permalink">Polymorphic forms of o-benzylphenol and slow molecular motions in the amorphous state
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>This paper is about the number 1. The number 2 is left for future work.</p>

</p>
    

    
      <p>Published in <i>Journal of Non-Crystalline Solids</i>, 2004 </p>
    

    
      <p>Download <a href=" http://www.sciencedirect.com/science/article/pii/S0022309304006271 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2008-21-04-paper2/" rel="permalink">Water‐Soluble Monofunctional Perylene and Terrylene Dyes: Powerful Labels for Single‐Enzyme Tracking
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
All in one: Exceptionally photostable, highly fluorescent, water‐soluble, and monofunctional perylene and terrylene dyes bearing reactive groups for covalent attachment to biomolecules have been synthesized (see picture). Single‐molecule enzyme tracking revealed that single enzymes could be visualized even on a substrate with fluorescent background.
</p>
</p>
    

    
      <p>Published in <i>Angewandte Chemie</i>, 2008 </p>
    

    
      <p>Download <a href=" https://onlinelibrary-wiley-com.kuleuven.ezproxy.kuleuven.be/doi/full/10.1002/anie.200705409 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2009-12-01-paper3/" rel="permalink">Linking Phospholipase Mobility to Activity by Single-Molecule Wide-Field Microscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p>First article reporting my work.</p>

</p>
    

    
      <p>Published in <i>ChemPhysChem</i>, 2009 </p>
    

    
      <p>Download <a href=" https://onlinelibrary-wiley-com.kuleuven.ezproxy.kuleuven.be/doi/full/10.1002/cphc.200800537 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2010-01-05-paper4/" rel="permalink">Unraveling Excited-State Dynamics in a Polyfluorene-Perylenediimide Copolymer
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Insight into the exciton dynamics occurring in a polyfluorene-perylenediimide (PF-PDI) copolymer with a reaction mixture ratio of 100 fluorene units to 1 N,N′-bis(phenyl)-1,6,7,12-tetra(p-tert-octylphenoxy)-perylene-3,4,9,10-tetracarboxylic acid diimide (PDI) is presented here. Time-correlated single photon counting and femtosecond transient absorption spectroscopy measurements on the PF-PDI copolymer have been employed to investigate the excited-state properties of the polyfluorene subunit where the exciton is localized (PF) and the incorporated PDI chromophore. The experimental results were compared with those obtained from a polyfluorene polymer (model PF) and a N,N′-bis(2,6-diisopropylphenyl)-1,6,7,12-tetra(p-tert-octylphenoxy)-perylene-3,4,9,10-tetracarboxylic acid diimide (model PDI) which were used as reference compounds. Because of the high polydispersity of the PF-PDI copolymer, there is a polymer fraction present that contains no PDI chromophores (polyfluorene polymer fraction (PF polymer fraction)), and wide-field imaging of single polymers chains of the synthesized PF-PDI copolymer was used to estimate this PF polymer fraction. Following the primary excitation of the PF in the PF-PDI copolymer, energy hopping between PF’s can occur. A fraction of the energy of the absorbed photons will be transferred to a PDI chromophore via energy transfer from a PF. In a polar solvent, a charge transfer state having the S1 of the PDI moiety as a precursor state is found to form with high efficiency on a nanosecond time scale. The data suggest that a fraction of the absorbed energy is directed, transferred, and used in charge separation, providing a clear view of a multistep mechanism of exciton dissociation into charges.
</p>
</p>
    

    
      <p>Published in <i>The Journal of Physical Chemistry B</i>, 2010 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=3145034621131176659&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2010-05-05-paper5/" rel="permalink">Influence of lipid heterogeneity and phase behavior on phospholipase A 2 action at the single molecule level
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
We monitored the action of phospholipase A2 (PLA2) on L- and D-dipalmitoyl-phosphatidylcholine (DPPC) Langmuir monolayers by mounting a Langmuir-trough on a wide-field fluorescence microscope with single molecule sensitivity. This made it possible to directly visualize the activity and diffusion behavior of single PLA2 molecules in a heterogeneous lipid environment during active hydrolysis. The experiments showed that enzyme molecules adsorbed and interacted almost exclusively with the fluid region of the DPPC monolayers. Domains of gel state L-DPPC were degraded exclusively from the gel-fluid interface where the buildup of negatively charged hydrolysis products, fatty acid salts, led to changes in the mobility of PLA2. The mobility of individual enzymes on the monolayers was characterized by single particle tracking. Diffusion coefficients of enzymes adsorbed to the fluid interface were between 3.2 μm2/s on the L-DPPC and 4.9 μm2/s on the D-DPPC monolayers. In regions enriched with hydrolysis products, the diffusion dropped to ≈0.2 μm2/s. In addition, slower normal and anomalous diffusion modes were seen at the L-DPPC gel domain boundaries where hydrolysis took place. The average residence times of the enzyme in the fluid regions of the monolayer and on the product domain were between ≈30 and 220 ms. At the gel domains it was below the experimental time resolution, i.e., enzymes were simply reflected from the gel domains back into solution.
</p>
</p>
    

    
      <p>Published in <i>Biophysical journal</i>, 2010 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=6878295774835300498&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2011-05-27-paper6/" rel="permalink">Fluorescent probes for superresolution imaging of lipid domains on the plasma membrane
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Accumulating evidence indicates that membrane lipids are not randomly distributed but rather form specific domains. In particular, raft-like microdomains composed of cholesterol and sphingolipids are attracting a lot of attention. These microdomains are thought to serve as platforms for signal transduction and molecular trafficking, but it is difficult to elucidate their detailed structure since their reported size is smaller than the resolution of light microscopy. To circumvent this limitation, we designed probes for cholesterol- and sphingolipid-enriched microdomains dedicated for superresolution microscopy, PALM. The probes utilise the affinity of the toxins, θ-toxin and lysenin, for the cholesterol- and sphingomyelin-enriched membranes, respectively. The toxicity can be avoided by using non-toxic domains that retain the specific binding to the aforementioned membranes. The probes can easily be produced in E. coli as recombinant protein domains of toxins fused to a photoswitchable fluorescent protein, Dronpa. PALM imaging with these probes revealed two types of cholesterol-enriched microdomains, line-shaped ones with widths of around 150 nm and round ones with an average radius of 118 nm. All sphingomyelin-enriched microdomains were round with an average radius of 124 nm. Both the cholesterol- and sphingomyelin-enriched microdomains vanished by the depletion of cholesterol. The sphingomyelin-enriched microdomains also vanished by the depletion of sphingomyelin whereas the cholesterol-enriched microdomains were unaffected. We conclude that cholesterol- and sphingomyelin-enriched domains occupy different regions on the plasma membrane.
</p>
</p>
    

    
      <p>Published in <i>Chemical Science</i>, 2011 </p>
    

    
      <p>Download <a href=" http://pubs.rsc.org/-/content/articlehtml/2011/sc/c1sc00169h "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2011-10-21-paper7/" rel="permalink">Local elongation of endothelial cell-anchored von Willebrand factor strings precedes ADAMTS13 protein-mediated proteolysis
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Platelet-decorated von Willebrand factor (VWF) strings anchored to the endothelial surface are rapidly cleaved by ADAMTS13. Individual VWF string characteristics such as number, location, and auxiliary features of the ADAMTS13 cleavage sites were explored here using imaging and computing software. By following changes in VWF string length, we demonstrated that VWF strings are cleaved multiple times, successively shortening string length in the function of time and generating fragments ranging in size from 5 to over 100 μm. These are larger than generally observed in normal plasma, indicating that further proteolysis takes place in circulation. Interestingly, in 89% of all cleavage events, VWF strings elongate precisely at the cleavage site before ADAMTS13 proteolysis. These local elongations are a general characteristic of VWF strings, independent of the presence of ADAMTS13. Furthermore, large elongations, ranging in size from 1.4 to 40 μm, occur at different sites in space and time. In conclusion, ADAMTS13-mediated proteolysis of VWF strings under flow is preceded by large elongations of the string at the cleavage site. These elongations may lead to the simultaneous exposure of many exosites, thereby facilitating ADAMTS13-mediated cleavage.
</p>
</p>
    

    
      <p>Published in <i>Journal of Biological Chemistry</i>, 2011 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=4741833987433377542&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2011-12-15-paper8/" rel="permalink">Quantitative multicolor super-resolution microscopy reveals tetherin HIV-1 interaction
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Virus assembly and interaction with host-cell proteins occur at length scales below the diffraction limit of visible light. Novel super-resolution microscopy techniques achieve nanometer resolution of fluorescently labeled molecules. The cellular restriction factor tetherin (also known as CD317, BST-2 or HM1.24) inhibits the release of human immunodeficiency virus 1 (HIV-1) through direct incorporation into viral membranes and is counteracted by the HIV-1 protein Vpu. For super-resolution analysis of HIV-1 and tetherin interactions, we established fluorescence labeling of HIV-1 proteins and tetherin that preserved HIV-1 particle formation and Vpu-dependent restriction, respectively. Multicolor super-resolution microscopy revealed important structural features of individual HIV-1 virions, virus assembly sites and their interaction with tetherin at the plasma membrane. Tetherin localization to micro-domains was dependent on both tetherin membrane anchors. Tetherin clusters containing on average 4 to 7 tetherin dimers were visualized at HIV-1 assembly sites. Combined biochemical and super-resolution analysis revealed that extended tetherin dimers incorporate both N-termini into assembling virus particles and restrict HIV-1 release. Neither tetherin domains nor HIV-1 assembly sites showed enrichment of the raft marker GM1. Together, our super-resolution microscopy analysis of HIV-1 interactions with tetherin provides new insights into the mechanism of tetherin-mediated HIV-1 restriction and paves the way for future studies of virus-host interactions.
</p>
</p>
    

    
      <p>Published in <i>PLoS pathogens</i>, 2011 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=8064940300684147224&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2012-03-01-paper9/" rel="permalink">Mapping of Surface‐Enhanced Fluorescence on Metal Nanoparticles using Super‐Resolution Photoactivation Localization Microscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Photoactivation localization microscopy (PALM) was applied to study surface‐enhanced fluorescence (SEF) on metal nanostructures (SEF‐PALM). The detection of fluorescence from individual single molecules can be used to image the point‐spread‐function and spatial distribution of the fluorescence emitted in the vicinity of a metal surface. Due to the strong scattering effect, the angular distribution of the fluorescence is altered by metals, resulting in a spatial shift of fluorescence spots with respect to the metal nanostructures, and has to be taken into account in the analysis. SEF‐PALM can be used to discriminate effects of labelling density when estimating the enhancement factor in SEF. Furthermore, nanostructures with sizes below the diffraction limit can be resolved using this technique. SEF‐PALM is established as a powerful tool to study plasmon‐mediated phenomena on metal nanostructures.
</p>
</p>
    

    
      <p>Published in <i>ChemPhysChem</i>, 2012 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=2117513180994007070&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2012-06-18-paper10/" rel="permalink">Spectroscopic properties, excitation, and electron transfer in an anionic water-soluble poly (fluorene-alt-phenylene)-perylenediimide copolymer
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
An anionic fluorene-phenylene poly{1,4-phenylene-[9,9-bis(4-phenoxy-butylsulfonate)]fluorene-2,7-diyl}-based copolymer containing on-chain perylenediimine (PDI) chromophoric units, PBS-PFP-PDI, was synthesized and its photophysical properties studied as aggregates and isolated chains in water and dioxane/water (1:1) solution. UV–vis and emission spectroscopy measurements, time-correlated single photon counting, and wide field imaging have been employed to investigate the excited-state behavior of the PBS-PFP-PDI copolymer, including the effect of environment on the energy and electron transfer to the on-chain PDI chromophore. Although the Förster overlap integral is favorable, no evidence is found for intramolecular singlet excitation energy transfer in isolated copolymer chains in solution. Fluorescence is suggested to involve an interchain process, thus revealing that isolated copolymer chains in solution do not undergo efficient intramolecular energy transfer. However, quenching of the PBS-PFP excited state by PDI is observed in aqueous media and ultrafast pump–probe studies in water or dioxane–water solutions show that electron transfer occurs from the phenylene-fluorene units to the PDI. The extent of electron transfer increases with aggregation, suggesting it is largely an interchain process. The interaction of the negatively charged PBS-PFP-PDI copolymer with the positively charged surfactant hexadecyltrimethylammonium bromide (CTAB) in solution has also been studied. The copolymer PBS-PFP-PDI aggregates with the surfactant already at concentrations below the critical micelle concentration (cmc) and the nonpolar environment allows intermolecular energy transfer, observed by the weak emission band located at 630 nm that is associated with the emission of the PDI chromophore. However, the fact that the PDI photoluminescence (PL) lifetime (∼1.4 ns) obtained in the presence of CTAB is considerably shorter than that of the nonaggregated chromophore (∼5.4 ns) suggests that even in this case there is considerable PL quenching, possibly through some charge transfer route. The increase of the PBS-PFP-PDI photoluminescence intensity at surfactant concentrations above the cmc indicates deaggregation of polyelectrolyte within the initially formed polyelectrolyte–surfactant aggregates.
</p>
</p>
    

    
      <p>Published in <i>The Journal of Physical Chemistry B</i>, 2012 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=15400515330618560664&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2012-12-31-paper11/" rel="permalink">Ensemble and single particle fluorimetric techniques in concerted action to study the diffusion and aggregation of the glycine receptor α3 isoforms in the cell plasma membrane
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
The spatio-temporal membrane behavior of glycine receptors (GlyRs) is known to be of influence on receptor homeostasis and functionality. In this work, an elaborate fluorimetric strategy was applied to study the GlyR α3K and L isoforms. Previously established differential clustering, desensitization and synaptic localization of these isoforms imply that membrane behavior is crucial in determining GlyR α3 physiology. Therefore diffusion and aggregation of homomeric α3 isoform-containing GlyRs were studied in HEK 293 cells. A unique combination of multiple diffraction-limited ensemble average methods and subdiffraction single particle techniques was used in order to achieve an integrated view of receptor properties. Static measurements of aggregation were performed with image correlation spectroscopy (ICS) and, single particle based, direct stochastic optical reconstruction microscopy (dSTORM). Receptor diffusion was measured by means of raster image correlation spectroscopy (RICS), temporal image correlation spectroscopy (TICS), fluorescence recovery after photobleaching (FRAP) and single particle tracking (SPT). The results show a significant difference in diffusion coefficient and cluster size between the isoforms. This reveals a positive correlation between desensitization and diffusion and disproves the notion that receptor aggregation is a universal mechanism for accelerated desensitization. The difference in diffusion coefficient between the clustering GlyR α3L and the non-clustering GlyR α3K cannot be explained by normal diffusion. SPT measurements indicate that the α3L receptors undergo transient trapping and directed motion, while the GlyR α3K displays mild hindered diffusion. These findings are suggestive of differential molecular interaction of the isoforms after incorporation in the membrane.
</p>
</p>
    

    
      <p>Published in <i>Biochimica et Biophysica Acta (BBA)-Biomembranes</i>, 2012 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=17311243176504726847&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2013-08-01-paper12/" rel="permalink">Role of PFKFB3-driven glycolysis in vessel sprouting
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Vessel sprouting by migrating tip and proliferating stalk endothelial cells (ECs) is controlled by genetic signals (such as Notch), but it is unknown whether metabolism also regulates this process. Here, we show that ECs relied on glycolysis rather than on oxidative phosphorylation for ATP production and that loss of the glycolytic activator PFKFB3 in ECs impaired vessel formation. Mechanistically, PFKFB3 not only regulated EC proliferation but also controlled the formation of filopodia/lamellipodia and directional migration, in part by compartmentalizing with F-actin in motile protrusions. Mosaic in vitro and in vivo sprouting assays further revealed that PFKFB3 overexpression overruled the pro-stalk activity of Notch, whereas PFKFB3 deficiency impaired tip cell formation upon Notch blockade, implying that glycolysis regulates vessel branching.
</p>
</p>
    

    
      <p>Published in <i>Cell</i>, 2013 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=6461457367768971277&amp;btnI=1&amp;hl=enA "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2013-12-01-paper13/" rel="permalink">EGF receptor dynamics in EGF-responding cells revealed by functional imaging during single particle tracking
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
The epidermal growth factor (EGF) receptor transduces the extracellular EGF signal into the cells. The distribution of these EGF receptors in the plasma membrane is heterogeneous and dynamic, which is proposed to be important for the regulation of cell signaling. The response of the cells to a physiological concentration of EGF is not homogeneous, which makes it difficult to analyze the dynamics related to the response. Here we developed a system to perform functional imaging during single particle tracking (SPT) analysis. This system made it possible to observe the cytosolic Ca2+ concentration to monitor the cell response while tracking individual EGF molecules and found that about half of the cells responded to the stimulation with 1.6 nM EGF. In the responding cells, the EGF receptor showed 3 modes of movement: fast (the diffusion coefficient of 0.081 ± 0.009 μm2/sec, 29 ± 9%), slow (0.020 ± 0.005 μm2/sec, 22 ± 6%), and stationary (49 ± 13%). The diffusion coefficient of the fast mode movement in the responding cells was significantly larger than that in the nonresponding cells (0.069 ± 0.009 μm2/sec, p &lt; 0.05). The diffusion coefficient of the fast mode movement is thought to reflect the monomer–dimer equilibrium of the EGF receptor. We assumed that the feedback regulation via the Ca2+ signaling pathway slightly shifts the equilibrium from dimer to monomer in the responding cells.
</p>
</p>
    

    
      <p>Published in <i>Biophysical Reviews and Letters</i>, 2013 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=15161066838312640196&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2013-12-02-paper14/" rel="permalink">Excited state dynamics of the photoconvertible fluorescent protein Kaede revealed by ultrafast spectroscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
The ultrafast excited state dynamics of the fluorescent protein Kaede has been investigated by employing time resolved fluorescence and transient absorption. Upon irradiation of its neutral state, the protein undergoes an efficient conversion to a state that fluoresces at longer wavelengths. The molecular basis of the photoconversion involves an expansion of the chromophore π-conjugation by formal β-elimination but details of the reaction pathway remain subject to debate. Based on the kinetics observed in experiments on the protein sample in both H2O and D2O buffers, we suggest that a light-initiated cleavage mechanism (20 ps) could take place, forming the neutral red state in which the red chromophore resides. Excitation of the neutral red form results in the formation of the red anionic species via two Förster resonance energy transfer (FRET) channels. FRET between red neutral and red anionic forms occurs within the tetramer with time constants of 13.4 ps and 210 ps. In contrast to literature proposals no ESPT was observed
</p>
</p>
    

    
      <p>Published in <i>Photochemical &amp; Photobiological Sciences</i>, 2013 </p>
    

    
      <p>Download <a href=" http://pubs.rsc.org/-/content/articlehtml/2014/pp/c3pp50335f "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-01-13-paper15/" rel="permalink">Cellular localization and dynamics of the Mrr type IV restriction endonuclease of Escherichia coli
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
In this study, we examined the intracellular whereabouts of Mrr, a cryptic type IV restriction endonuclease of Escherichia coli K12, in response to different conditions. In absence of stimuli triggering its activity, Mrr was found to be strongly associated with the nucleoid as a number of discrete foci, suggesting the presence of Mrr hotspots on the chromosome. Previously established elicitors of Mrr activity, such as exposure to high (hydrostatic) pressure (HP) or expression of the HhaII methyltransferase, both caused nucleoid condensation and an unexpected coalescence of Mrr foci. However, although the resulting Mrr/nucleoid complex was stable when triggered with HhaII, it tended to be only short-lived when elicited with HP. Moreover, HP-mediated activation of Mrr typically led to cellular blebbing, suggesting a link between chromosome and cellular integrity. Interestingly, Mrr variants could be isolated that were specifically compromised in either HhaII- or HP-dependent activation, underscoring a mechanistic difference in the way both triggers activate Mrr. In general, our results reveal that Mrr can take part in complex spatial distributions on the nucleoid and can be engaged in distinct modes of activity.
</p>
</p>
    

    
      <p>Published in <i>Nucleic Acids Research</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=9567037385680422925&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-01-21-paper16/" rel="permalink">Green-to-red photoconvertible Dronpa mutant for multimodal super-resolution fluorescence microscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Advanced imaging techniques crucially depend on the labels used. In this work, we present the structure-guided design of a fluorescent protein that displays both reversibly photochromic and green-to-red photoconversion behavior. We first designed ffDronpa, a mutant of the photochromic fluorescent protein Dronpa that matures up to three times faster while retaining its interesting photochromic features. Using a combined evolutionary and structure-driven rational design strategy, we developed a green-to-red photoconvertible ffDronpa mutant, called pcDronpa, and explored different optimization strategies that resulted in its improved version, pcDronpa2. This fluorescent probe combines a high brightness with low photobleaching and photoblinking. We herein show that, despite its tetrameric nature, pcDronpa2 allows for multimodal subdiffraction imaging by sequentially imaging a given sample using both super-resolution fluctuation imaging and localization microscopy.
</p>
</p>
    

    
      <p>Published in <i>ACS Nano</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=1298372096764935190&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-02-01-paper17/" rel="permalink">A surface-bound molecule that undergoes optically biased Brownian rotation
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Developing molecular systems with functions analogous to those of macroscopic machine components, such as rotors, gyroscopes and valves, is a long-standing goal of nanotechnology. However, macroscopic analogies go only so far in predicting function in nanoscale environments, where friction dominates over inertia. In some instances, ratchet mechanisms have been used to bias the ever-present random, thermally driven (Brownian) motion and drive molecular diffusion in desired directions. Here, we visualize the motions of surface-bound molecular rotors using defocused fluorescence imaging, and observe the transition from hindered to free Brownian rotation by tuning medium viscosity. We show that the otherwise random rotations can be biased by the polarization of the excitation light field, even though the associated optical torque is insufficient to overcome thermal fluctuations. The biased rotation is attributed instead to a fluctuating-friction mechanism in which photoexcitation of the rotor strongly inhibits its diffusion rate.
</p>
</p>
    

    
      <p>Published in <i>Nature Nanotechnology</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=4768184005452428216&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-03-28-paper18/" rel="permalink">Single particle tracking of ADAMTS13 (a disintegrin and metalloprotease with thrombospondin type-1 repeats) molecules on endothelial von Willebrand factor strings
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
von Willebrand factor (VWF) strings are removed from the endothelial surface by ADAMTS13 (a disintegrin and metalloprotease with thrombospondin type-1 repeats)-mediated proteolysis. To visualize how single ADAMTS13 molecules bind to these long strings, we built a customized single molecule fluorescence microscope and developed single particle tracking software. Extensive analysis of over 6,000 single inactive ADAMTS13E225Q enzymes demonstrated that 20% of these molecules could be detected in at least two consecutive 60-ms frames and followed two types of trajectories. ADAMTS13E225Q molecules either decelerated in the vicinity of VWF strings, whereas sometimes making brief contact with the VWF string before disappearing again, or readily bound to the VWF strings and this for 120 ms or longer. These interactions were observed at several sites along the strings. Control experiments using an IgG protein revealed that only the second type of trajectory reflected a specific interaction of ADAMTS13 with the VWF string. In conclusion, we developed a dedicated single molecule fluorescence microscope for detecting single ADAMTS13 molecules (nm scale) on their long, flow-stretched VWF substrates (μm scale) anchored on living cells. Comprehensive analysis of all detected enzymes showed a random interaction mechanism for ADAMTS13 with many available binding sites on the VWF strings.
</p>
</p>
    

    
      <p>Published in <i>Journal of Biological Chemistry</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=17279771921569505920&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-03-31-paper19/" rel="permalink">Analysis of α3 GlyR single particle tracking in the cell membrane
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Single particle tracking (SPT) of transmembrane receptors in the plasma membrane often reveals heterogeneous diffusion. A thorough interpretation of the displacements requires an extensive analysis suited for discrimination of different motion types present in the data. Here the diffusion pattern of the homomeric α3-containing glycine receptor (GlyR) is analyzed in the membrane of HEK 293 cells. More specifically, the influence of the α3 RNA splice variants α3K and α3L on lateral membrane diffusion of the receptor is revealed in detail. Using a combination of ensemble and local SPT analysis, free and anomalous diffusion parameters are determined. The GlyR α3 free diffusion coefficient is found to be 0.13 ± 0.01 μm2/s and both receptor variants display confined motion. The confinement probability level and residence time are significantly elevated for the α3L variant compared to the α3K variant. Furthermore, for the α3L GlyR, the presence of directed motion was also established, with a velocity matching that of saltatory vesicular transport. These findings reveal that α3 GlyRs are prone to different types of anomalous diffusion and reinforce the role of RNA splicing in determining lateral membrane trafficking.
</p>
</p>
    

    
      <p>Published in <i>Biochimica et Biophysica Acta (BBA)-Molecular Cell Research</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=2414249255948516963&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-04-02-paper20/" rel="permalink">HIV virions as nanoscopic test tubes for probing oligomerization of the integrase enzyme
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Employing viruses as nanoscopic lipid-enveloped test tubes allows the miniaturization of protein–protein interaction (PPI) assays while preserving the physiological environment necessary for particular biological processes. Applied to the study of the human immunodeficiency virus type 1 (HIV-1), viral biology and pathology can also be investigated in novel ways, both in vitro as well as in infected cells. In this work we report on an experimental strategy that makes use of engineered HIV-1 viral particles, to allow for probing PPIs of the HIV-1 integrase (IN) inside viruses with single-molecule Förster resonance energy transfer (FRET) using fluorescent proteins (FP). We show that infectious fluorescently labeled viruses can be obtained and that the quantity of labels can be accurately measured and controlled inside individual viral particles. We demonstrate, with proper control experiments, the formation of IN oligomers in single viral particles and inside viral complexes in infected cells. Finally, we show a clear effect on IN oligomerization of small molecule inhibitors of interactions of IN with its natural human cofactor LEDGF/p75, corroborating that IN oligomer enhancing drugs are active already at the level of the virus and strongly suggesting the presence of a dynamic, enhanceable equilibrium between the IN dimer and tetramer in viral particles. Although applied to the HIV-1 IN enzyme, our methodology for utilizing HIV virions as nanoscopic test tubes for probing PPIs is generic, i.e., other PPIs targeted into the HIV-1, or PPIs targeted into other viruses, can potentially be studied with a similar strategy.
</p>
</p>
    

    
      <p>Published in <i>ACS Nano</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=10448916991829412725&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-04-05-paper21/" rel="permalink">Membrane remodeling processes induced by phospholipase action
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Important cellular events such as division require drastic changes in the shape of the membrane. These remodeling processes can be triggered by the binding of specific proteins or by changes in membrane composition and are linked to phospholipid metabolism for which dedicated enzymes, named phospholipases, are responsible. Here wide-field fluorescence microscopy is used to visualize shape changes induced by the action of phospholipase A1 on dye-labeled supported membranes of POPC (1-palmitoyl-2-oleoly-sn-glycero-3-phosphocholine). Time-lapse imaging demonstrates that layers either shrink and disappear or fold and collapse into vesicles. These vesicles can undergo further transformations such as budding, tubulation, and pearling within 5 min of formation. Using dye-labeled phospholipases, we can monitor the presence of the enzyme at specific positions on the membrane as the shape transformations occur. Furthermore, incorporating the products of hydrolysis into POPC membranes is shown to induce transformations similar to those observed for enzyme action. The results suggest that phospholipase-mediated hydrolysis plays an important role in membrane transformations by altering the membrane composition, and a model is proposed for membrane curvature based on the presence and shape of hydrolysis products.
</p>
</p>
    

    
      <p>Published in <i>Langmuir</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=5578343296461734504&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-07-01-paper22/" rel="permalink">Membrane distribution of the glycine receptor α3 studied by optical super-resolution microscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
In this study, the effect of glycine receptor (GlyR) α3 alternative RNA splicing on the distribution of receptors in the membrane of human embryonic kidney 293 cells is investigated using optical super-resolution microscopy. Direct stochastic optical reconstruction microscopy is used to image both α3K and α3L splice variants individually and together using single- and dual-color imaging. Pair correlation analysis is used to extract quantitative measures from the resulting images. Autocorrelation analysis of the individually expressed variants reveals clustering of both variants, yet with differing properties. The cluster size is increased for α3L compared to α3K (mean radius 92 ± 4 and 56 ± 3 nm, respectively), yet an even bigger difference is found in the cluster density (9,870 ± 1,433 and 1,747 ± 200 μm−2, respectively). Furthermore, cross-correlation analysis revealed that upon co-expression, clusters colocalize on the same spatial scales as for individually expressed receptors (mean co-cluster radius 94 ± 6 nm). These results demonstrate that RNA splicing determines GlyR α3 membrane distribution, which has consequences for neuronal GlyR physiology and function.
</p>
</p>
    

    
      <p>Published in <i>Histochemistry and Cell Biology</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=13476617484225502928&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-08-01-paper23/" rel="permalink">Live‐Cell SERS Endoscopy Using Plasmonic Nanowire Waveguides
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Live‐cell surface‐enhanced Raman spectroscopy (SERS) endoscopy is developed by using plasmonic nanowire waveguides as endoscopic probes. It is demonstrated that the probe insertion does not stress the cell. Opposed to conventional SERS endoscopy, with excitation at the hotspot within the cell, the remote excitation method yields low‐background SERS spectra from specific cell compartments with minimal associated photodamage.
</p>
</p>
    

    
      <p>Published in <i>Advanced Materials</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=4007959408368814304&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2014-09-05-paper24/" rel="permalink">The HIV-1 integrase mutant R263A/K264A is 2-fold defective for TRN-SR2 binding and viral nuclear import
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Transportin-SR2 (Tnpo3, TRN-SR2), a human karyopherin encoded by the TNPO3 gene, has been identified as a cellular cofactor of HIV-1 replication, specifically interacting with HIV-1 integrase (IN). Whether this interaction mediates the nuclear import of HIV remains controversial. We previously characterized the TRN-SR2 binding interface in IN and introduced mutations at these positions to corroborate the biological relevance of the interaction. The pleiotropic nature of IN mutations complicated the interpretation. Indeed, all previously tested IN interaction mutants also affected reverse transcription (RT). Here we report on a virus with a pair of IN mutations, INR263A/K264A, that significantly reduce interaction with TRN-SR2. The virus retains wild-type reverse transcription activity but displays a block in nuclear import and integration, as measured by Q-PCR. The defect in integration of this mutant resulted in a smaller increase in the number of 2-LTR circles than for virus specifically blocked at integration by raltegravir or catalytic site mutations (IND64N/D116N/E152Q). Finally, using an eGFP-IN labeled HIV fluorescence-based import assay, the defect in nuclear import was corroborated. These data altogether underscore the importance of the HIV-IN TRN-SR2 protein-protein interaction for HIV nuclear import and validate the IN/TRN-SR2 interaction interface as a promising target for future antiviral therapy.
</p>
</p>
    

    
      <p>Published in <i>Journal of Biological Chemistry</i>, 2014 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=13138577458701614229&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2015-09-01-paper25/" rel="permalink">Super-resolution mapping of glutamate receptors in C. elegans by confocal correlated PALM
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Photoactivated localization microscopy (PALM) is a super-resolution imaging technique based on the detection and subsequent localization of single fluorescent molecules. PALM is therefore a powerful tool in resolving structures and putative interactions of biomolecules at the ultimate analytical detection limit. However, its limited imaging depth restricts PALM mostly to in vitro applications. Considering the additional need for anatomical context when imaging a multicellular organism, these limitations render the use of PALM in whole animals difficult. Here we integrated PALM with confocal microscopy for correlated imaging of the C. elegans nervous system, a technique we termed confocal correlated PALM (ccPALM). The neurons, lying below several tissue layers, could be visualized up to 10 μm deep inside the animal. By ccPALM, we visualized ionotropic glutamate receptor distributions in C. elegans with an accuracy of 20 nm, revealing super-resolution structure of receptor clusters that we mapped onto annotated neurons in the animal. Pivotal to our results was the TIRF-independent detection of single molecules, achieved by genetic regulation of labeled receptor expression and localization to effectively reduce the background fluorescence. By correlating PALM with confocal microscopy, this platform enables dissecting biological structures with single molecule resolution in the physiologically relevant context of whole animals.
</p>
</p>
    

    
      <p>Published in <i>Scientific Reports</i>, 2015 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=2003348855037362207&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2015-02-17-paper26/" rel="permalink">Visualization of molecular fluorescence point spread functions via remote excitation switching fluorescence microscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
The enhancement of molecular absorption, emission and scattering processes by coupling to surface plasmon polaritons on metallic nanoparticles is a key issue in plasmonics for applications in (bio)chemical sensing, light harvesting and photocatalysis. Nevertheless, the point spread functions for single-molecule emission near metallic nanoparticles remain difficult to characterize due to fluorophore photodegradation, background emission and scattering from the plasmonic structure. Here we overcome this problem by exciting fluorophores remotely using plasmons propagating along metallic nanowires. The experiments reveal a complex array of single-molecule fluorescence point spread functions that depend not only on nanowire dimensions but also on the position and orientation of the molecular transition dipole. This work has consequences for both single-molecule regime-sensing and super-resolution imaging involving metallic nanoparticles and opens the possibilities for fast size sorting of metallic nanoparticles, and for predicting molecular orientation and binding position on metallic nanoparticles via far-field optical imaging.
</p>
</p>
    

    
      <p>Published in <i>Nature Communications</i>, 2015 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=16442790511705799301&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2015-11-13-paper27/" rel="permalink">Mechanism behind the apparent large Stokes shift in LSSmOrange investigated by time-resolved spectroscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
LSSmOrange is a fluorescent protein with a large energy gap between the absorption and emission bands (5275 cm–1). The electronic structure of the LSSmOrange chromophore, 2-[(5-)-2-hydroxy-dihydrooxazole]-4-(p-hydroxybenzylidene)-5-imidazolinone, is affected by deprotonation of the p-hydroxybenzylidene group. We investigated LSSmOrange by time-resolved spectroscopy in the femtosecond and nanosecond range. The ground state chromophore was almost exclusively in the neutral form, which had a main absorption band at 437 nm with a small shoulder at 475 nm. The absorption at a wavelength within the former band promoted the protein to the excited state where excited state proton transfer (ESPT) could lead to deprotonation in 0.8 ps. Following ESPT, the chromophore emitted fluorescence with a maximum at 573 nm and a decay time of 3500 ps. Although deprotonation by ESPT occurs, we unexpectedly found a slow accumulation of the anionic form in the ground state upon repeated high intensity excitation. This accumulation of the anionic form was accompanied by a shift of the absorption band to 553 nm without changing the emission band. MALDI-MS revealed that this shift is accompanied by decarboxylation of E222, which is interacting with the imidazolinone ring of the chromophore. We concluded that the photoinduced decarboxylation induced a conformational change that affected local environment around the hydroxyl group, resulting in a stable deprotonated form of the chromophore.
</p>
</p>
    

    
      <p>Published in <i>The Journal of Physical Chemistry B</i>, 2015 </p>
    

    
      <p>Download <a href=" http://pubs.acs.org/doi/abs/10.1021/acs.jpcb.5b09189 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2016-01-29-paper29/" rel="permalink">Super-resolution localization and defocused fluorescence microscopy on resonantly coupled single-molecule, single-nanorod hybrids
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Optical antennas made of metallic nanostructures dramatically enhance single-molecule fluorescence to boost the detection sensitivity. Moreover, emission properties detected at the optical far field are dictated by the antenna. Here we study the emission from molecule–antenna hybrids by means of super-resolution localization and defocused imaging. Whereas gold nanorods make single-crystal violet molecules in the tip’s vicinity visible in fluorescence, super-resolution localization on the enhanced molecular fluorescence reveals geometrical centers of the nanorod antenna instead. Furthermore, emission angular distributions of dyes linked to the nanorod surface resemble that of nanorods in defocused imaging. The experimental observations are consistent with numerical calculations using the finite-difference time-domain method.
</p>
</p>
    

    
      <p>Published in <i>ACS nano</i>, 2016 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=12275411927469986018&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2016-11-10-paper28/" rel="permalink">Dynamic Oligomerization of Integrase Orchestrates HIV Nuclear Entry
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Nuclear entry is a selective, dynamic process granting the HIV-1 pre-integration complex (PIC) access to the chromatin. Classical analysis of nuclear entry of heterogeneous viral particles only yields averaged information. We now have employed single-virus fluorescence methods to follow the fate of single viral pre-integration complexes (PICs) during infection by visualizing HIV-1 integrase (IN). Nuclear entry is associated with a reduction in the number of IN molecules in the complexes while the interaction with LEDGF/p75 enhances IN oligomerization in the nucleus. Addition of LEDGINs, small molecule inhibitors of the IN-LEDGF/p75 interaction, during virus production, prematurely stabilizes a higher-order IN multimeric state, resulting in stable IN multimers resistant to a reduction in IN content and defective for nuclear entry. This suggests that a stringent size restriction determines nuclear pore entry. Taken together, this work demonstrates the power of single-virus imaging providing crucial insights in HIV replication and enabling mechanism-of-action studies.
</p>
</p>
    

    
      <p>Published in <i>Scientific reports</i>, 2016 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=16369962557174192966&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2016-09-06-paper30/" rel="permalink">Photoconvertible behavior of LSSmOrange applicable for single emission band optical highlighting
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Photoswitchable fluorescent proteins are capable of changing their spectral properties upon light irradiation, thus allowing one to follow a chosen subpopulation of molecules in a biological system. Recently, we revealed a photoinduced absorption band shift of LSSmOrange, which was originally engineered to have a large energy gap between excitation and emission bands. Here, we evaluated the performance of LSSmOrange as a fluorescent tracer in living cells. The absorption maximum of LSSmOrange in HeLa cells shifted from 437 nm to 553 nm upon illumination with a 405-, 445-, 458-, or 488-nm laser on a laser-scanning microscope, whereas the emission band remained same (∼570 nm). LSSmOrange behaves as a freely diffusing protein in living cells, enabling the use of the protein as a fluorescence tag for studies of protein dynamics. By targeting LSSmOrange in mitochondria, we observed an exchange of soluble molecules between the matrices upon mitochondrial fusion. Since converted and unconverted LSSmOrange proteins have similar emission spectra, this tracer offers unique possibilities for multicolor imaging. The fluorescence emission from LSSmOrange was spectrally distinguishable from that of eYFP and mRFP, and could be separated completely by applying linear unmixing. Furthermore, by using a femtosecond laser at 850 nm, we showed that a two-photon process could evoke a light-induced red shift of the absorption band of LSSmOrange, providing a strict confinement of the conversion volume in a three-dimensional space.
</p>
</p>
    

    
      <p>Published in <i>Biophysical journal</i>, 2016 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=15499433308006697399&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2017-03-02-paper31/" rel="permalink">The ER stress sensor PERK coordinates ER-plasma membrane contact site formation through interaction with filamin-A and F-actin remodeling
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Loss of ER Ca2+ homeostasis triggers endoplasmic reticulum (ER) stress and drives ER-PM contact sites formation in order to refill ER-luminal Ca2+. Recent studies suggest that the ER stress sensor and mediator of the unfolded protein response (UPR) PERK regulates intracellular Ca2+ fluxes, but the mechanisms remain elusive. Here, using proximity-dependent biotin identification (BioID), we identified the actin-binding protein Filamin A (FLNA) as a key PERK interactor. Cells lacking PERK accumulate F-actin at the cell edges and display reduced ER-PM contacts. Following ER-Ca2+ store depletion, the PERK-FLNA interaction drives the expansion of ER-PM juxtapositions by regulating F-actin-assisted relocation of the ER-associated tethering proteins Stromal Interaction Molecule 1 (STIM1) and Extended Synaptotagmin-1 (E-Syt1) to the PM. Cytosolic Ca2+ elevation elicits rapid and UPR-independent PERK dimerization, which enforces PERK-FLNA-mediated ER-PM juxtapositions. Collectively, our data unravel an unprecedented role of PERK in the regulation of ER-PM appositions through the modulation of the actin cytoskeleton.
</p>
</p>
    

    
      <p>Published in <i>Molecular Cell</i>, 2017 </p>
    

    
      <p>Download <a href=" https://scholar.google.be/scholar?oi=bibs&amp;cluster=12463296102564186802&amp;btnI=1&amp;hl=en "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2017-19-12-paper32/" rel="permalink">Orthogonal Probing of Single Molecule Heterogeneity by Correlative Fluorescence and Force Microscopy
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Correlative imaging by fluorescence and force microscopy is an emerging technology to acquire orthogonal information at the nanoscale. Whereas atomic force microscopy excels at resolving the envelope structure of nanoscale specimens, fluorescence microscopy can detect specific molecular labels, which enables the unambiguous recognition of molecules in a complex assembly. Whereas correlative imaging at the micrometer scale has been established, it remains challenging to push the technology to the single-molecule level. Here, we used an integrated setup to systematically evaluate the factors that influence the quality of correlative fluorescence and force microscopy. Optimized data processing to ensure accurate drift correction and high localization precision results in image registration accuracies of ∼25 nm on organic fluorophores, which represents a 2-fold improvement over the state of the art in correlative fluorescence and force microscopy. Furthermore, we could extend the Atto532 fluorophore bleaching time ∼2-fold, by chemical modification of the supporting mica surface. In turn, this enables probing the composition of macromolecular complexes by stepwise photobleaching with high confidence. We demonstrate the performance of our method by resolving the stoichiometry of molecular subpopulations in a heterogeneous EcoRV–DNA nucleoprotein ensemble.
</p>
</p>
    

    
      <p>Published in <i>ACS nano</i>, 2017 </p>
    

    
      <p>Download <a href=" https://pubs.acs.org/doi/10.1021/acsnano.7b05405 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2018-03-13-paper33/" rel="permalink">Photoconversion of Far‐Red Organic Dyes: Implications for Multicolor Super‐Resolution Imaging
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Localization‐based super‐resolution microscopy has become an indispensable tool in biology to study features smaller than the diffraction limit of light. In a multicolor approach, adequate spectral separation of the different photoswitchable probes is required. A far‐red emitting dye is often one of the labels of choice. However, irradiation with high laser intensity can induce photo‐conversion of some of the most frequently used fluorophores. Herein we show that upon intense irradiation with a 561 nm laser line, far‐red organic dyes photoconvert to blue‐shifted emissive species. In the case of Alexa Fluor 647, the most commonly used fluorescent label in super‐resolution microscopy, this derivative is created over time in an intramolecular, irreversible photoinduced chemical reaction. The dynamics of this reaction are altered by the presence of reducing agents. Importantly, the blue‐shifted derivatives emit in the spectral range of the red fluorescent proteins (e. g. PAmCherry and converted mEos3.2), severely implicating multicolor super‐resolution imaging.
</p>
</p>
    

    
      <p>Published in <i>ChemPhotoChem</i>, 2018 </p>
    

    
      <p>Download <a href=" https://onlinelibrary.wiley.com/doi/full/10.1002/cptc.201700216 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2018-05-13-paper34/" rel="permalink">Single Viruses on the Fluorescence Microscope: Imaging Molecular Mobility, Interactions and Structure Sheds New Light on Viral Replication
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Viruses are simple agents exhibiting complex reproductive mechanisms. Decades of research have provided crucial basic insights, antiviral medication and moderately successful gene therapy trials. The most infectious viral particle is, however, not always the most abundant one in a population, questioning the utility of classic ensemble-averaging virology. Indeed, viral replication is often not particularly efficient, prone to errors or containing parallel routes. Here, we review different single-molecule sensitive fluorescence methods that we employ routinely to investigate viruses. We provide a brief overview of the microscopy hardware needed and discuss the different methods and their application. In particular, we review how we applied (i) single-molecule Förster resonance energy transfer (smFRET) to probe the subviral human immunodeficiency virus (HIV-1) integrase (IN) quaternary structure;(ii) single particle tracking to study interactions of the simian virus 40 with membranes;(iii) 3D confocal microscopy and smFRET to quantify the HIV-1 pre-integration complex content and quaternary structure;(iv) image correlation spectroscopy to quantify the cytosolic HIV-1 Gag assembly, and finally;(v) super-resolution microscopy to characterize the interaction of HIV-1 with tetherin during assembly. We hope this review is an incentive for setting up and applying similar single-virus imaging studies in daily virology practice.
</p>
</p>
    

    
      <p>Published in <i>Viruses</i>, 2018 </p>
    

    
      <p>Download <a href=" http://www.mdpi.com/1999-4915/10/5/250 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2018-09-07-paper35/" rel="permalink">Mapping Transient Protein Interactions at the Nanoscale in Living Mammalian Cells
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Protein–protein interactions (PPIs) form the basis of cellular processes, regulating cell behavior and fate. PPIs can be extremely transient in nature, which hinders their detection. In addition, traditional biochemical methods provided limited information on the spatial distribution and temporal dynamics of PPIs that is crucial for their regulation in the crowded cellular environment. Given the pivotal role of membrane micro- and nanodomains in the regulation of PPIs at the plasma membrane, the development of methods to visualize PPIs with a high spatial resolution is imperative. Here, we present a super-resolution fluorescence microscopy technique that can detect and map short-lived transient protein–protein interactions on a nanometer scale in the cellular environment. This imaging method is based on single-molecule fluorescence microscopy and exploits the effect of the difference in the mobility between cytosolic and membrane-bound proteins in the recorded fluorescence signals. After the development of the proof of concept using a model system based on membrane-bound modular protein domains and fluorescently labeled peptides, we applied this imaging approach to investigate the interactions of cytosolic proteins involved in the epidermal growth factor signaling pathway (namely, Grb2, c-Raf, and PLCγ1). The detected clusters of Grb2 and c-Raf were correlated with the distribution of the receptor at the plasma membrane. Additionally, the interactions of wild type PLCγ1 were compared with those detected with truncated mutants, which provided important information regarding the role played by specific domains in the interaction with the membrane. The results presented here demonstrate the potential of this technique to unravel the role of membrane heterogeneity in the spatiotemporal regulation of cell signaling.
</p>
</p>
    

    
      <p>Published in <i>ACS Nano</i>, 2018 </p>
    

    
      <p>Download <a href=" https://pubs.acs.org/doi/10.1021/acsnano.8b01227 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2018-09-08-paper36/" rel="permalink">Role of glutamine synthetase in angiogenesis beyond glutamine synthesis
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Glutamine synthetase, encoded by the gene GLUL, is an enzyme that converts glutamate and ammonia to glutamine. It is expressed by endothelial cells, but surprisingly shows negligible glutamine-synthesizing activity in these cells at physiological glutamine levels. Here we show in mice that genetic deletion of Glul in endothelial cells impairs vessel sprouting during vascular development, whereas pharmacological blockade of glutamine synthetase suppresses angiogenesis in ocular and inflammatory skin disease while only minimally affecting healthy adult quiescent endothelial cells. This relies on the inhibition of endothelial cell migration but not proliferation. Mechanistically we show that in human umbilical vein endothelial cells GLUL knockdown reduces membrane localization and activation of the GTPase RHOJ while activating other Rho GTPases and Rho kinase, thereby inducing actin stress fibres and impeding endothelial cell motility. Inhibition of Rho kinase rescues the defect in endothelial cell migration that is induced by GLUL knockdown. Notably, glutamine synthetase palmitoylates itself and interacts with RHOJ to sustain RHOJ palmitoylation, membrane localization and activation. These findings reveal that, in addition to the known formation of glutamine, the enzyme glutamine synthetase shows unknown activity in endothelial cell migration during pathological angiogenesis through RHOJ palmitoylation.
</p>
</p>
    

    
      <p>Published in <i>Nature</i>, 2018 </p>
    

    
      <p>Download <a href=" https://www.nature.com/articles/s41586-018-0466-7 "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2019-02-01-paperIndra/" rel="permalink">Organoids from pituitary as a novel research model toward pituitary stem cell exploration
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
The pituitary is the master endocrine gland, harboring stem cells of which the phenotype and role remain poorly characterized. Here, we established organoids from mouse pituitary with the aim to generate a novel research model to study pituitary stem cell biology. The organoids originated from the pituitary cells expressing the stem cell marker SOX2 were long-term expandable, displayed a stemness phenotype during expansive culture and showed specific hormonal differentiation ability, although limited, after subrenal transplantation. Application of the protocol to transgenically injured pituitary harboring an activated stem cell population, resulted in more numerous organoids. Intriguingly, these organoids presented with a cystic morphology, whereas the organoids from undamaged gland were predominantly dense and appeared more limited in expandability. Transcriptomic analysis revealed distinct epithelial phenotypes and showed that cystic organoids more resembled the pituitary phenotype, at least to an immature state, and displayed in vitro differentiation, although yet moderate. Organoid characterization further exposed facets of regulatory pathways of the putative stem cells of the pituitary and advanced new injury-activated markers. Taken together, we established a novel organoid research model revealing new insights into the identity and regulation of the putative pituitary stem cells. This organoid model may eventually lead to an interesting tool to decipher pituitary stem cell biology in both healthy and diseased gland.
</p>
</p>
    

    
      <p>Published in <i>Journal of Endocrinology</i>, 2019 </p>
    

    
      <p>Download <a href=" https://joe.bioscientifica.com/view/journals/joe/240/2/JOE-18-0462.xml "><u>here</u></a></p>
    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/publications/2019-02-01-paper37/" rel="permalink">Polymeric Engineering of Nanoparticles for Highly Efficient Multifunctional Drug Delivery Systems
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><h2> Abstract </h2>
<p align="justify">
Most targeting strategies of anticancer drug delivery systems (DDSs) rely on the surface functionalization of nanocarriers with specific ligands, which trigger the internalization in cancer cells via receptor-mediated endocytosis. The endocytosis implies the entrapment of DDSs in acidic vesicles (endosomes and lysosomes) and their eventual ejection by exocytosis. This process, intrinsic to eukaryotic cells, is one of the main drawbacks of DDSs because it reduces the drug bioavailability in the intracellular environment. The escape of DDSs from the acidic vesicles is, therefore, crucial to enhance the therapeutic performance at low drug dose. To this end, we developed a multifunctionalized DDS that combines high specificity towards cancer cells with endosomal escape capabilities. Doxorubicin-loaded mesoporous silica nanoparticles were functionalized with polyethylenimine, a polymer commonly used to induce endosomal rupture, and hyaluronic acid, which binds to CD44 receptors, overexpressed in cancer cells. We show irrefutable proof that the developed DDS can escape the endosomal pathway upon polymeric functionalization. Interestingly, the combination of the two polymers resulted in higher endosomal escape efficiency than the polyethylenimine coating alone. Hyaluronic acid additionally provides the system with cancer targeting capability and enzymatically controlled drug release. Thanks to this multifunctionality, the engineered DDS had cytotoxicity comparable to the pure drug whilst displaying high specificity towards cancer cells. The polymeric engineering here developed enhances the performance of DDS at low drug dose, holding great potential for anticancer therapeutic applications.
</p>
</p>
    

    
      <p>Published in <i>Scientific reports</i>, 2019 </p>
    

    
      <p>Download <a href=" https://www.nature.com/articles/s41598-019-39107-3#article-info "><u>here</u></a></p>
    

  </article>
</div>

<h2>talks</h2>

<h2>teaching</h2>

<h2>team</h2>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Susana/" rel="permalink">Susana Rocha
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p><span style="color:gray; fontangle:italic;">August 2019</span>)</p>

</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="1901-01-01T00:00:00+01:00">January 01, 1901</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Monica/" rel="permalink">Monica Ricci
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
In 2006 researchers found a new way to ‘reprogram’ adult specialized cells into stem cells. These so-called induced pluripotent stem cells (iPSCs) were a breakthrough for genetic studies in human cells. The ability to genetically modify human iPSCs holds great promise for gene therapy and other clinical applications. However, while the immortalized human tumour cells lines commonly use at the laboratory level can be edited with almost complete efficiency, much lower success rates are achieved in iPSCs. Using the recent advances in nanotechnology, we propose to develop a new method for the delivery of genetic material in the cellular environment with high delivery efficiency and applicable to all cell types, including stem cells.
It has been already shown that nanoscale structures, such as nanotubes and nanowires (NWs), have the ability to penetrate cell membranes with minimal cellular damage. Such nanotechnology provides a powerful new modality for delivering genetic material and other biomolecules and has opened up new opportunities to manipulate living cells. Using NW-based gene delivery the amount of DNA, critical for genetic modification, is easily controlled.
The application of NW based gene delivery to reprogram adult cells into iPSCs, modify its genome and induce differentiation into a specific cell type has the potential to drive autologous gene therapy a step closer to clinical applications.


</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2015-10-01T00:00:00+02:00">October 01, 2015</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Danai/" rel="permalink">Danai  Laskaratou
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Cells use special proteins called receptors to communicate with the extracellular environment and respond to the incoming signals accordingly. In this project we plan to study ErbB3, which belongs to the ErbB family of four receptors mediating normal cell growth, differentiation, development and cell death. The family members can interact with each other and form pairs, either between two different type of receptors (heterodimers) or between the same type of receptors (homodimers). The different pairings can lead to the activation of various signaling molecules and pathways. ErbB3 in particular forms heterodimers with other family
receptors and initiates a signaling pathway which prevents programmed cell death. Defective ErbB receptors have been found in several cancer types, but the role of ErbB3 in cancer has been recognized only recently. Here we aim to reveal the mechanisms governing the dimer formation and the cell signaling that follows. The results of the project will not only contribute to a better understanding of signaling in general, but also set a foundation for novel therapeutic applications for cancer.
</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2016-10-01T00:00:00+02:00">October 01, 2016</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Boris/" rel="permalink">Boris Louis
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Polymers are large molecules formed by linkage of repetitive structural units leading to a molecular chain and, more recently, to sheet-like structures. Polymers properties are determined not only by their structural units but also by their conformation and environment, thus, offering great possibilities to tune their properties. Polymer materials are flexible and light while they can be stronger than steel, they can conduct electricity and emit light. Consequently, they take an important part in our modern world. Moreover, even for the same material, individual molecules differ from one another due to the different conformations they can adopt. Therefore, to better understand the world of polymers, we propose to study individual molecules rather than bulk samples in which such individual level information is averaged out. We will use fluorescence microscopy and spectroscopy to study some of the most remarkable polymers available today: conjugated polymers used in highly efficient solar cell, dendronized polymers (DP) and two- dimensional sheet-like polymers (2DP). Connecting the electronic properties of conjugated chains to their conformation is crucial for fundamental physical understanding, paving the way for improvement in opto-electronic devices. The DPs will allow us to study polymer reptation in 3D and compare it to the reptation theory. Finally, the 2DPs are interesting as a new, unexplored materials where virtually nothing is known about their potential applications

</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-01-01T00:00:00+01:00">January 01, 2017</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Laurens/" rel="permalink">Laurens D’Huys
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">

</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-09-08T00:00:00+02:00">September 08, 2017</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Indra/" rel="permalink">Indra van Zundert
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Cancer is one of the major causes of death in our society. The anti-cancer drugs currently used in chemotherapy produce several side effects and, although fatal for almost all cells in a tumor, a small percentage of cells appear to resist the treatment. Therefore, it is urgent to design new therapies that specifically target these cells while causing no harm to healthy tissue. The resistance to multiple drugs is linked to the presence of specific molecules at the cellular plasma membrane, that actively pump out chemical drugs (efflux pumps). Multi-drug resistant cells are suggested to be the main cause of treatment failure and relapse.
In recent years, nanoparticle-based drug delivery platforms have emerged as the next-generation of pharmaceutical compounds. The surface of the nanoparticles can be modified to target specific cells, while the core of the particle can be loaded with anti-cancer drugs. In this project we want to go one step further and develop multifunctional nanoparticles to attack multi-drug resistant cells. In addition to loading the particles with anti-cancer drug(s) and targeting them to cancer cells, we will supply them with tools to reduce the amount of efflux pumps at the cell membrane. This can be achieved using cutting-edge gene editing methodologies. The versatility of the resulting nanoparticles can be infinite.

</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-10-01T00:00:00+02:00">October 01, 2017</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Johannes/" rel="permalink">Johannes Vandaele
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Organ failure and tissue lost are challenging health issues due to the lack of organs for transplantation and the limitations of conventional artificial implants. Tissue engineering has emerged as a promising approach to generate biological substitutes. In tissue engineering applications, cells are grown in a three-dimensional platform which is known as a scaffold.
In the last years a great effort has been made in the development of new synthetic biomimetic materials to be used as scaffolds in tissue engineering. Recent studies have shown that the structure and mechanical properties of the materials play a crucial role in regulating cell behaviour. The adjustable mechanical properties of synthetic materials provide more control over the induced cellular behaviours. However, there is lack of technologies that can investigate the role of mechanical and structural properties at the subcellular and molecular level.
In this project I will develop new microscopic approaches to investigate both the structure and mechanics of hydrogels at the nano- and micrometer scale, and their influence in cellular behaviours. As a model, polyisocyanopeptide (PIC) gels will be used. This unique material resembles naturally occurring polymers, while maintaining tunable properties. However, the microscopic toolbox developed will be easily applied to other materials. The fundamental knowledge acquired will be crucial to develop the next generation of biomimetic materials for tissue engineering.
</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2017-11-01T00:00:00+01:00">November 01, 2017</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Quinten/" rel="permalink">Quinten Coucke
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Mechanical forces play an undisputed elementary role in the interactions between cells and the surrounding extracellular matrix (ECM). Not only are these forces essential for the cell migratory behavior, they also influence proliferation (including tumor growth) and differentiation. These forces are transferred across focal adhesions (FAs) which connect ECM and cell skeleton through patches of activated integrin proteins. Since the origin of exerted cellular forces lies in these FAs, they are the ideal starting point for characterization of mechanotransduction pathways.
Consequently, studying how the properties of the ECM affect the cellular forces is key in understanding how cells connect to their environment and alter their behavior appropriately.  While the scientific field of cellular mechanosensation has been studied for years, recent developments in imaging techniques and force sensor development enable us to dig deeper.
A home-built confocal Fluorescence Lifetime Imaging Microscopy (FLIM) microscope is used to measure Förster Resonance Energy Transfer (FRET) in FAs.

</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2018-10-01T00:00:00+02:00">October 01, 2018</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Hongbo/" rel="permalink">Hongbo Yuan
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description"><p align="justify">
Cells continually sense the external forces and generate internal forces within 3D extracellular matrix (ECM), such force, associated with focal adhesions (FAs) and cytoskeleton, stiffens and remodels their surrounding environment in response to specific mechanical cues. However, it is extremely difficult to carry out systematic studies either with natural ECM or non-fibrillar synthetic hydrogels, because of less controllable parameters and lack of biological relevance, respectively. To avoid these problems, we will investigate the bi-directional force interactions in biomimetic strain-stiffening matrix, polyisocyanides (PIC) hydrogels, which allow us to control their stiffness, critical stress, and ligand density independently, developed by Rowan/Kouwer's lab in Radboud University Nijmegen, where I finished my PhD. Therefore, the goal of this action is to study the FAs involved force transmission between cell and matrix at nano-scale by the cutting-edge fluorescence microscopy techniques in the host institution, KU Leuven. We will be the first to link the matrix properties, size and distribution of FAs, cytoskeletal structures and cell morphology together by the multicolour 3D super-resolution optical fluctuation imaging (SOFI) technique. The forces generated by cells will be quantified both on cellular and single FA level by traction force microscope (TFM) and molecular tension sensor-based FRET. In addition, we will study how cells remodel matrix in a physical manner, including polymer network reorganization and stiffening mechanical properties. These results will have a high impact for understanding how cells interact with matrix through force.
</p>
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-01-01T00:00:00+01:00">January 01, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Charlotte/" rel="permalink">Charlotte Cresens
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description">
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-10-01T00:00:00+02:00">October 01, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Sergey/" rel="permalink">Sergey Abakumov
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description">
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-10-02T00:00:00+02:00">October 02, 2019</time></p>
    

    

  </article>
</div>

<!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

<div class="list__item">
  <article class="archive__item" itemscope="" itemtype="http://schema.org/CreativeWork">
    

    <h2 class="archive__item-title" itemprop="headline">
      
        <a href="http://localhost:4000/team/Mathias/" rel="permalink">Mathias Fraiponts
</a>
      
    </h2>

    



    
    <p class="archive__item-excerpt" itemprop="description">
</p>
    

    
     <p class="page__date"><strong><i class="fa fa-fw fa-calendar" aria-hidden="true"></i> Published:</strong> <time datetime="2019-10-03T00:00:00+02:00">October 03, 2019</time></p>
    

    

  </article>
</div>

<h2>techniques</h2>


  </div>
</div>


    </script>

    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<a href="/news/">News</a>
<!-- end custom footer snippets -->

        


<div class="page__footer-copyright">&copy; 2019 Susana Rocha's Lab. Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

<script type="text/javascript">try{ clicky.init(101142812); }catch(e){}</script>
<noscript><p><img alt="Clicky" width="1" height="1" src="//in.getclicky.com/101142812ns.gif" /></p></noscript>
</body>
</html>

      </footer>
    </div>

    <script src="http://localhost:4000/assets/js/main.min.js"></script>



    <!-- Jekyll Ideal Image Slider Include -->
<!-- https://github.com/jekylltools/jekyll-ideal-image-slider-include -->
<!-- v1.8 -->

  </body>
</html>

